llvm/test/CodeGen/AMDGPU/fneg-combines.new.ll

```diff
@@ -3440,6 +3485,6 @@ declare half @llvm.amdgcn.rcp.f16(half) #1
 
 attributes #0 = { nounwind "denormal-fp-math-f32"="preserve-sign,preserve-sign" }
 attributes #1 = { nounwind readnone }
-attributes #2 = { nounwind "unsafe-fp-math"="true" }
+attributes #2 = { nounwind }
```

Comment:
This is evidently relying on other properties, probably should leave this for now (or find the minimum set of attributes + flags to preserve the old codegen). I see this at least was using "no-signed-zeros-fp-math"

Reply:
`"no-signed-zeros-fp-math"="true"` is enough here and re-adding `"unsafe-fp-math"="true"` changes nothing in this pull request.